src/client.ts

```diff
@@ -587,7 +589,7 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
    * @returns a promise that fullfils once the mute request has completed
    */
   async setAudioMute (muteOptions: ISessionMuteRequest): Promise<void> {
-    !muteOptions.fromHeadset && this.headset.setMute(muteOptions.mute);
+    this.headset.setMute(muteOptions.mute);
     await this.sessionManager.setAudioMute(muteOptions);
```

Comment:
As I'm looking at this again, is it going to cause issues since we aren't awaiting the `headset.setMute()` promise? I'm on the fence about it because we don't want to _not_ call `sessionmanager.setAudioMute()` if the headset call fails. But at the same time, callers won't have the ability to catch the headset error.